python/paddle/quantization/__init__.py

```diff
@@ -12,35 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...fluid.contrib.slim.quantization.imperative.ptq_config import (
+from ..fluid.contrib.slim.quantization.imperative.ptq_config import (
```

Comment:
If these classes/functions `ptq_config`(and below) need to be used, it is recommended to migrate their implementation here, and the fluid is also cleaning.

Reply:
Fluid will be removed in another PR: https://github.com/PaddlePaddle/Paddle/pull/48717

Reply:
ok, thanks